src/controllers/dlgprefcontroller.cpp

```diff
@@ -367,8 +367,9 @@ void DlgPrefController::slotUpdate() {
     m_ui.outputMappingsTab->setEnabled(isMappable);
 }
 
-void DlgPrefController::slotCancel() {
-    slotShowPreset(m_pController->getPreset());
+void DlgPrefController::slotResetToDefaults() {
+    m_ui.chkEnabledDevice->setChecked(false);
```

Comment:
This still leaves the description of the old preset showing. Manually selecting the "No preset" item works as expected.

Reply:
Fixed. It will also ask you if you want to save/discard the unsaved preset changes and mark the dialog as dirty so that the values will be saved when you hit apply.